geode-connectors/src/main/java/org/apache/geode/connectors/jdbc/internal/AbstractJdbcCallback.java

```diff
@@ -35,11 +35,7 @@ protected AbstractJdbcCallback(SqlHandler sqlHandler, InternalCache cache) {
   }
 
   @Override
-  public void close() {
-    if (sqlHandler != null) {
-      sqlHandler.close();
-    }
-  }
+  public void close() {}
```

Comment:
The default is "{}" so you can just delete close() {}.

Reply:
done